data_models/models.py

```diff
@@ -34,6 +34,8 @@ class Meta:
 
     about = models.TextField('Om Radio Revolt')
 
+    privacy_policy = models.TextField('Personvernerklæring', default="")
```

Comment:
This field should be added to the snapshot test in api_graphql/tests.py in the "test_settings" test. (Line 72)
Should also add chiefEditor and radioEditor to the same query :innocent: